### PR TITLE
:robot: Allow testing immucore from branches

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:

Just allows to build immucore from a given branch and inject it into the final iso for easy testing

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -36,6 +36,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{fromJson(needs.get-matrix.outputs.matrix)}}
+    env:
+      - IMMUCORE_DEV: true
```
Comment (mudler) [inline .github/workflows/image.yaml:40]: how does this work?? is an `.env` file created by github automatically? from what I knew you could use a single env for earthly https://docs.earthly.dev/docs/guides/build-args#overriding-argument-values

Comment (Itxaka) [reply .github/workflows/image.yaml]: not even working locally, not my best day :D

**Review by mudler — APPROVED** [review]
Nice!

**Comment by Itxaka** [comment]
I don't really want to merge this without having a dispatcher or something to manually trigger it.

I did not really expected it to pass this soon 😝

**Comment by Itxaka** [comment]
now it should allow to trigger it manually and provide the parameters

**Review by mudler — DISMISSED** [review]

**Pull request merged**